action-processor/create-action.js

```diff
@@ -340,6 +345,24 @@ async function createMarketingAuthorisation (featureName, action) {
     localStorage.setItem(featureName, 'maList', maList)
 }
 
+async function createMarketingAuthorisationApplication (featureName, action) {
+    log.debug(`${SERVICE_NAME}::createMarketingAuthorisationApplication`)
+    let maData = action.data
+    log.info(`${SERVICE_NAME}::createMarketingAuthorisationApplication::${action.label}::creating marketing authorisation application from ${JSON.stringify(maData)}`)
+    let responseData = await maApplicationService.createMarketingAuthorisationApplication(maData)
+    log.info(`${SERVICE_NAME}::createMarketingAuthorisationApplication::${action.label}::created:${JSON.stringify(responseData)}`)
+    var savedAction = localStorage.getItem(featureName, action.label)
+    savedAction.response = responseData
+    log.debug(`${SERVICE_NAME}::createMarketingAuthorisationApplication, saved action ${JSON.stringify(savedAction)}`)
+    localStorage.setItem(featureName, action.label, savedAction)
+    var maList = localStorage.getItem(featureName, 'maList')
```

Comment:
Can we change the key name to `maApplicationList` please as it is the same as the marketing authorisation test data list.